index.d.ts

```diff
@@ -27,6 +27,8 @@ export type AfterResponseHook = (
 	response: Response,
 ) => Response | void | Promise<Response | void>;
 
+export type ParseJson = <T = unknown>(text: string) => T
```

Comment:
I don't think it needs an exported type. Can just put the type inline.

Reply:
I'm not sure I fully understand the use of the type variable `T` here. What's the benefit over just `unknown`?

Reply:
> I'm not sure I fully understand the use of the type variable `T` here. What's the benefit over just `unknown`?

Fair, will make it just `unknown`

Reply:
fixed